### PR TITLE
Add .nvmrc file to specify node version

### PR DESCRIPTION
**Pull request by petele**
Quickly select the right version of node when using `nvm` for the repo.

### CODE REVIEW

**Comment by ddbeck** [comment]
Huh, does nvm support `.tool-versions`? I use asdf for this and have it ignored, but it would be nice to check in something like it.

(I'll see if asdf could use .nvmrc too.)

**Comment by petele** [comment]
Interesting - I don't use asdf, and am not familiar with it. But, it doesn't look like it does. :(

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+22
```
Comment (ddbeck) [inline .nvmrc:1]: OK, I couldn't find a way to consolidate on a single file. How about we just set this to be consistent with `.node-version` and be done with it?

```suggestion
18
```

Comment (petele) [reply .nvmrc:1]: I tried running it with 18.16.1 and get an error that I don't get with 22.4.1. @tropicadri mentioned she was having problems with 18 as well and that 22 worked for her. I'm OK with going with 18, but we'll need to specify a subversion as well. 

Which version are you running?

```
file:///Users/petele/Sites/web-features/packages/compute-baseline/dist/browser-compat-data/compat.js:1
import bcd from "@mdn/browser-compat-data" with { type: "json" };
                                           ^^^^

SyntaxError: Unexpected token 'with'
    at ESMLoader.moduleStrategy (node:internal/modules/esm/translators:119:18)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:468:14)

Node.js v18.16.1
```

Comment (ddbeck) [reply .nvmrc:1]: Ah ha. It needs to be >=18.20, which is when import attributes landed in Node.js.

**Review by ddbeck — APPROVED** [review]

**Pull request merged**